packages/lit-dev-content/samples/examples/react-basics/demo-greeting.ts

```diff
@@ -8,7 +8,21 @@ export class DemoGreeting extends LitElement {
   render() {
     return html`
       <p>Hello, ${this.name}!</p>
-      <p>Checkout our <a href="https://lit.dev/docs/frameworks/react/" target="_blank" rel="noopener noreferrer">React framework docs</a>!</p>
+      <p>
+        Checkout our
+        <a
+          href="https://lit.dev/docs/frameworks/react/"
+          target="_blank"
+          rel="noopener noreferrer"
+          >React framework docs</a
```

Comment:
I worry a little bit that any mention of React in this component might make it seem like this component was specifically written to work in React, even though that's not the case. Just thinking from the POV of someone completely unfamiliar with Lit - maybe they're a React dev told to check this out by someone who says you can use Lit in React.

If really we want a message on screen here, we could put it in index.html or app.tsx